blueocean-core-js/src/js/utils/UrlUtils.js

```diff
@@ -276,6 +276,12 @@ export function toClassicJobPage(pageUrl, isMultibranch = false) {
         const fullJobNameTokens = fullJobName.split('/');
         const classicJobFullName = '/job/' + fullJobNameTokens.join('/job/');
 
+        // map to job only after there is token after pipelines. For example /organizations/jenkins/pipelines/p1
+        // can be mapped to /job/p1 but /organizations/jenkins/pipelines/ should be mapped to classic dashboard '/'
+        if (fullJobName === 'pipelines' && pageUrlTokens.length === 0) {
+            return undefined;
+        }
```

Comment:
yes, there were 2 regressions: https://issues.jenkins-ci.org/browse/JENKINS-44432 && https://issues.jenkins-ci.org/browse/JENKINS-44433 that this is addressing.

It was broken on master but also slightly broken before.